java/middleware/registry-middleware/authorization/src/main/java/dev/sunbirdrc/registry/authorization/SchemaAuthFilter.java

```diff
@@ -12,7 +12,7 @@
 
 public class SchemaAuthFilter implements Filter {
     private static final Logger logger = LoggerFactory.getLogger(SchemaAuthFilter.class);
-    private static final String INVITE_URL_ENDPOINT = "/invite";
+    private static final String INVITE_URL_PATTERN = "/api/v1/([A-Za-z0-9_])+/invite(/)?";
```

Comment:
Why is this changed.

Reply:
Added this to check whether the request uri is an invite endpoint. Earlier it was checking for contains "invite" which might not be well for a few cases